### PR TITLE
[4.1] Install from Web css and invisible badges

### PR DESCRIPTION
**Pull request by brianteeman**
This PR corrects a bug with the border radius of the search box in rtl (as shown below) and completely removes all the RTL specific css and uses logical properties instead.

The markup uses badge-color but bs5 only has bg-color so the badges were invisible. Also the hover on the card was white on white and that was invisible before

Also removes a few bits of css that are just not needed at all

## RTL before
image

## RTL after
image

## Badges & Hover Before
image

## Badges & Hover After
image

### CODE REVIEW

File: build/media_source/plg_installer_webinstaller/scss/client.scss [hunk 1]
```diff
@@ -55,10 +55,6 @@ $extension-types: (
     .card {
       height: 400px;
       margin-bottom: 2rem;
-
-      &:hover {
```
Comment (Quy) [inline build/media_source/plg_installer_webinstaller/scss/client.scss:59]: This should not be removed in order to have a visual change on hover. The card should have background color in order to see the keys (C M P).

Comment (brianteeman) [reply build/media_source/plg_installer_webinstaller/scss/client.scss:59]: thanks for that. I coudnt see it at all on my screen.

Comment (Quy) [reply build/media_source/plg_installer_webinstaller/scss/client.scss:59]: I changed it to grey from white to highlight that it should not be deleted.

Comment (brianteeman) [reply build/media_source/plg_installer_webinstaller/scss/client.scss:59]: the real problem is that the markup is using badge-color and that has been removed and replaced with bg-color. Will update this pr with a fix for both later

Comment (brianteeman) [reply build/media_source/plg_installer_webinstaller/scss/client.scss:59]: updated the PR and the original desceription

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c8bf92e3c244f404e02c379d81f37761e8813667<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36547">issues.joomla.org/tracker/joomla-cms/36547</a>.</sub>

**Comment by PhilETaylor** [comment]
I have tested this item :white_check_mark: successfully on c8bf92e3c244f404e02c379d81f37761e8813667<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36547">issues.joomla.org/tracker/joomla-cms/36547</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36547">issues.joomla.org/tracker/joomla-cms/36547</a>.</sub>

**Comment by brianteeman** [comment]
rebased to 4.1

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
thanks

**Comment by simbus82** [comment]
Thanks @brianteeman, this solve part of point 2 of this issue ;-)
https://github.com/joomla/install-from-web-server/issues/63

**Pull request merged**